app/app.go

```diff
@@ -242,7 +243,7 @@ func NewInitApp(
 	)
 
 	// Engine's module keepers
-	app.ownershipKeeper = ownershipsdk.NewKeeper(keys[ownershipsdk.ModuleName])
+	app.ownershipKeeper = ownership.NewKeeper(app.cdc, keys[ownership.ModuleName])
```

Comment:
the ownership module should declare its own `StoreKey` like other cosmos module. Eg:
```suggestion
	app.ownershipKeeper = ownership.NewKeeper(app.cdc, keys[ownership.StoreKey])
```